eslint.config.js

```diff
@@ -0,0 +1,43 @@
+const enactConfig = require('eslint-config-enact/strict');
+const importPlugin = require('eslint-plugin-import');
+const prettierPlugin = require('eslint-plugin-prettier');
+const prettierRecommended = require('eslint-config-prettier');
```

Comment:
you mentioned "but added prettier/recommended rules manually and it seems to work (source: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration)."
Is this duplicate import still needed?
Otherwise, let's just define `"const prettier = require('eslint-plugin-prettier');"`